front-end/src/app/layout/layout.component.ts

```diff
@@ -18,14 +18,29 @@ export enum BackgroundStyles {
   templateUrl: './layout.component.html',
   styleUrls: ['./layout.component.scss'],
 })
-export class LayoutComponent extends DestroyerComponent implements OnInit {
+export class LayoutComponent extends DestroyerComponent implements OnInit, AfterViewChecked {
   @ViewChild(FeedbackOverlayComponent) feedbackOverlay!: FeedbackOverlayComponent;
 
   layoutControls = new LayoutControls();
 
   constructor(private router: Router) {
     super();
   }
+  ngAfterViewChecked(): void {
+    this.updateContentOffset();
+  }
+
+  updateContentOffset() {
+    const contentOffset = document.getElementById('content-offset');
```

Comment:
Using document goes outside the Angular framework.
Would something like this work?

```
constructor(protected elementRef: ElementRef) ...

(<HTMLElement>this.elementRef.nativeElement).querySelector('#content-offset')
```

Reply:
Updated to use angular's renderer since I was already using the renderer in the Login Component. Updated it everywhere we were using document.getElementById which was just these two places and the DotFecService